packages/babel-helper-environment-visitor/src/index.ts

```diff
@@ -47,4 +46,4 @@ export default {
   ClassProperty(path) {
     skipAndrequeueComputedKeysAndDecorators(path);
   },
-} as Visitor<PluginPass>;
+} as Visitor<any>;
```

Comment:
Does `Visitor<unknown>` work here? If it does, it's safer since it doesn't "disable" type checking.